helpers/localizePrice.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const common = require('./lib/common.js');
+
+const factory = globals => {
+    return function(price, locale) {
+
+        if (!common.isObject(price) || !common.isString(price.currency) || isNaN(price.value)) {
+            // Return empty string if this does not appear to be a price object
+            return ''
```

Comment:
should we return original data without modification here instead?..

Reply:
In this case the assumption is that the theme developer is passing us something incorrect (a completely invalid object, a product image, something else) so we typically "fail" to give the theme developer some clue that they've done something wrong - and protect ourselves against attacks and stuff. That's the pattern in other helpers, but we don't have to follow it here.

I do consider prices to be "mission critical" for rendering a page, so perhaps a behavior that always tries to return something intelligible is better.

@mattolson @junedkazi interested in your perspective.

Reply:
👍 to failing over blind defaults

Reply:
Agree that if a user calls `localizePrice` with an invalid price object, we should fail. In the handlebars helper world, that means returning empty string (for consistency with behavior of other helpers, for example https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/money.js#L25).

Reply:
Need semicolon